### ISSUE TITLE
Documentation/comment code style rules

### ISSUE THREAD

**Issue by slava77** [cms-sw/cms-sw.github.io#101]
https://cmsdoxygen.web.cern.ch/cmsdoxygen/ is apparently still maintained/supported

What is the general feeling to still keep supporting the doxygen style comments in the code?

I frequently recall to ask for it in the DataFormats headers. (to use `///` or `//!`)
However, I do not see it mentioned in the https://cms-sw.github.io/cms_coding_rules.html

Should this be formalized in the rules?

**Comment by slava77** [comment]
@smuzaffar @makortel 
please clarify if this is considered.

**Comment by slava77** [comment]
ping

**Comment by smuzaffar** [comment]
Yes cmsdoxygen is still ailve but my general feeling is that LXR and DXR has replaced it. Regardless of we keep doxygen alive or not, it is true that we never had any clear rule/recommandation for documentation/comment style. The [coding rules](https://cms-sw.github.io/cms_coding_rules.html) were initially drived from the [1998 CMS Style rules](http://cds.cern.ch/record/687034/files/note98_072.pdf?version=1)  which is also not recomanding any comment style. 

I agree, we should add some guide lines for documentation/comments style in our code rules.
